sub-packages/bionemo-esm2/src/bionemo/esm2/model/finetune/dataset.py

```diff
@@ -123,6 +160,46 @@ def __getitem__(self, index: int) -> BertSample:
             "is_random": torch.zeros_like(tokenized_sequence, dtype=torch.int64),
         }
 
+    def _tokenize_with_special_tracking(self, sequence: str) -> tuple[Tensor, list[int]]:
```

Comment:
I would suggest to add a unit test for this

Reply:
done, thanks!